Week_7/homework/backend/routes/todo.js

```diff
@@ -0,0 +1,8 @@
+/** Module for handling users */
+const express = require("express");
+const cors = require("cors");
+
+// Define route and middlewares
+const users = express.Router();
+users.use(cors());
```

Comment:
Could you keep the todo routes in the todo.js file? I think you moved them so you could protect the route via authMiddleware, but you can export the authMiddleware in `auth.js` and import the auth middleware function here, then apply the middleware to the specified functions.